build_defs/test_helpers.bzl

```diff
@@ -231,3 +231,70 @@ def output_rule_diff_test(
         data = [expected_contents_csv, ":resultFact"],
         srcs = ["//src/analysis/souffle/tests/arcs_fact_tests:diff_wrapper.sh"],
     )
+    
+def run_sensitivity_analysis_compare_results(
+        name,
+        test_bin,
+        input_files,
+        visibility = None):
```

Comment:
This looks largely like a copy of the version for taint analysis. What are the differences? Is it possible to refactor this so that they share most of their code?